lightly/utils/benchmarking/metric_callback.py

```diff
@@ -58,8 +60,4 @@ def _append_metrics(
         self, metrics_dict: Dict[str, List[float]], trainer: Trainer
     ) -> None:
         for name, value in trainer.callback_metrics.items():
-            # Only store scalar values.
-            if isinstance(value, float) or (
-                isinstance(value, Tensor) and value.numel() == 1
-            ):
-                metrics_dict.setdefault(name, []).append(float(value))
+            metrics_dict.setdefault(name, []).append(float(value))
```

Comment:
I am not 100% sure if we can make this change because this code is called from code that is not yet type-checked and I am not sure if all metrics are really floats. There might also be tensors in there with more than 1 element which cannot be converted to floats. That's why we did the `value.numel() == 1` check.